src/templates/website/index.html

```diff
@@ -0,0 +1,88 @@
+{% from "_macros/app_tile.html" import app_tile %}
+
+{% set endpoint = api('website', [pk]) %}
+
+{% defer (url=endpoint, key=pk, id='website-data') %}
+  <section class="main">
+    <div class="subheader">
+      <h1>&nbsp;</h1>
+    </div>
+  </section>
+{% except %}
+  <section class="main">
+    <div class="subheader">
+      <h1>{{ _('Oh no!') }}</h1>
+    </div>
+  </section>
+  <section class="main full"><div>
+    {% if error == 403 %}
```

Comment:
I just copy pasted from the app page. Maybe I should leave the strings the same?